pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -51,6 +51,7 @@ message Schema {
     required bytes schema_data = 3;
     required Type type = 4;
     repeated KeyValue properties = 5;
+    optional bool is_auto_consume_schema = 6 [default = false];
```

Comment:
I think it's better to add the field to `CommandSubscribe` instead of the `Schema` itself.

BTW, please modify your PIP first before pushing the commits.